mne/io/edf/edf.py

```diff
@@ -114,6 +114,11 @@ def _read_segment_file(self, data, idx, fi, start, stop, cals, mult):
         annotmap = self._raw_extras[fi]['annotmap']
         subtype = self._raw_extras[fi]['subtype']
 
+        # Interpolating stim chan except it later gets overwritten.
+        if ((not (annot and annotmap or tal_channel is not None) and
+                (stim_channel in sel))):
+            warn('Interpolating stim channel. Events may jitter.')
```

Comment:
If I move it to `_get_edf_info`, I don't have access to `tal_channel` and `sel`.

All this is a bit cryptic to me, I'm just moving code around..